install.sh

```diff
@@ -115,10 +115,15 @@ if [ "${USERADD_ARGS}" != "" ]
 then
     echo "USERADD_ARGS=\"${USERADD_ARGS}\"" >> /etc/aws-ec2-ssh.conf
 fi
-
-sed -i 's:#AuthorizedKeysCommand none:AuthorizedKeysCommand /opt/authorized_keys_command.sh:g' /etc/ssh/sshd_config
-sed -i 's:#AuthorizedKeysCommandUser nobody:AuthorizedKeysCommandUser nobody:g' /etc/ssh/sshd_config
-
+condition=$(grep -i AuthorizedKeysCommand /etc/ssh/sshd_config | wc -l)
```

Comment:
grep with the -i flag checks case insensitive. Also, you did not include the # and the 'none' that is being used in the sed command.
And you also did not check for the AuthorizedKeysCommandUser line.

While this will work in some cases, it will break for example if the sshd_config has:

```
# We can use authorizedkeyscommand setting to run a script
# See the manual page for more info
AuthorizedKeysFile = /etc/ssh/authorized_keys
```

Better to check without the -i flag, and check the complete line we replace with sed.